packages/beacon-node/src/network/core/networkCoreWorkerHandler.ts

```diff
@@ -107,7 +107,10 @@ export class WorkerNetworkCore implements INetworkCore {
       loggerOpts: modules.logger.toOpts(),
     };
 
-    const worker = new Worker("./networkCoreWorker.js", {workerData} as ConstructorParameters<typeof Worker>[1]);
+    const worker = new Worker("./networkCoreWorker.js", {
+      workerData,
+      resourceLimits: {maxYoungGenerationSizeMb: 512},
```

Comment:
What's the default value?

Reply:
It's a [calculated value](https://github.com/nodejs/node/blob/4166d40d0873b6d8a0c7291872c8d20dc680b1d7/deps/v8/src/heap/heap.cc#L266) and ends up roughly 16mb with a `--max-old-space-size` of 4GB.